src/gpl/test/simple09.ok

```diff
@@ -39,6 +39,6 @@
 [INFO GPL-0028] BinCnt: 16 16
 [INFO GPL-0029] BinSize: 3872 3850
 [INFO GPL-0030] NumBins: 256
-[NesterovSolve] Iter: 1 overflow: 0 HPWL: 10073
+[NesterovSolve] Iter: 1 overflow: 0 HPWL: 116507
```

Comment:
This change doesn't work well and this is the symptom.   The isnan() code in NesterovPlace::init does not fire.  I think you are better off with something like:
```
@@ -540,7 +540,9 @@ int NesterovPlace::doNesterovPlace(int start_iter)
 
       debugPrint(log_, GPL, "np", 1, "NewStepLength: {:g}", newStepLength);
 
-      if (isnan(newStepLength) || isinf(newStepLength)) {
+      if (isinf(newStepLength)) {
+        newStepLength = 0;
+      } else if (isnan(newStepLength)) {
         isDiverged_ = true;
         divergeMsg_ = "RePlAce diverged at newStepLength.";
         divergeCode_ = 305;
```

Reply:
I tried to implement above changes, but unable to compile it. Error follows:
```
/home/vijayan/OPENROAD_FLOW/OpenROAD_FLOW/OpenROAD/src/gpl/src/nesterovPlace.cpp: In member function \u2018int gpl::NesterovPlace::doNesterovPlace(int)\u2019:
/home/vijayan/OPENROAD_FLOW/OpenROAD_FLOW/OpenROAD/src/gpl/src/nesterovPlace.cpp:544:11: error: \u2018isinif\u2019 was not declared in this scope
       if (isinif(newStepLength)) {
           ^~~~~~
/home/vijayan/OPENROAD_FLOW/OpenROAD_FLOW/OpenROAD/src/gpl/src/nesterovPlace.cpp:544:11: note: suggested alternative: \u2018isinff\u2019
       if (isinif(newStepLength)) {
           ^~~~~~
           isinff
gmake[2]: *** [src/gpl/CMakeFiles/gpl.dir/build.make:159: src/gpl/CMakeFiles/gpl.dir/src/nesterovPlace.cpp.o] Error 1
```

Reply:
I have updated with above format, but still got same logs.